app.js

```diff
@@ -95,100 +74,78 @@ function displayWeather(data) {
     <p>${data.temperature}</p>
   </div>
 
-  `
+  `;
 
   // update DOM
+
   weatherSection.firstElementChild.innerHTML = html;
 
 }
 
-
 // /* API request */
 // fetch('https://goweather.herokuapp.com/weather/spain')
 //   .then(response => response.json())
 //   .then(data => displayWeather(data));
 
-
-
-
-
-
-
-
 /*----------------------------------------------------*\
   Road Goat - retrieves interesting info about cities
 \*----------------------------------------------------*/
 
-
-
-
-
-
-
-
-
 /*----------------------------------------------------*\
   APP FUNCTIONS
 \*----------------------------------------------------*/
 
-
 // Function to render the user city selection on the page
 function displayCityName(inputCity) {
-
   // Create a paragraph to display the city name
+
   inputCityName = `<P>${inputCity}</P>`.toUpperCase();
   
   // Display the city name
   userInputDisplay.innerHTML = inputCityName; 
 
+  // Display the city name
+  userInputDisplay.append(inputCityName);
 }
 
-
 // Function to update ALL data on the page relative to the city
 function updateCityData() {
-
   // Retrieve value entered by user in input field
-  const inputCity = document.querySelector('#user-input__input--city').value
+  const inputCity = document.querySelector("#user-input__input--city").value;
 
   // Call displayCityName() to display the city name
   displayCityName(inputCity);
 
+  /* Giphy */
 
+  fetch(
+    `https://api.giphy.com/v1/gifs/search?q=${inputCity}&api_key=${giphyKey}`
+  )
+    .then(response => response.json())
+    // .then(data => console.log(data.data[0].images.downsized.url))
+    .then(data => displayImage(data));
 
+  /* Weather */
 
-/* Giphy */
-
-fetch(`https://api.giphy.com/v1/gifs/search?q=${inputCity}&api_key=${giphyKey}`)
-  .then(response => response.json())
-// .then(data => console.log(data.data[0].images.downsized.url))
-  .then(data => displayImage(data))
-
-
-
-/* Weather */
-
-/* API request */
-fetch(`https://goweather.herokuapp.com/weather/${inputCity}`)
-  .then(response => response.json())
-  .then(data => displayWeather(data));
+  /* API request */
+  fetch(`https://goweather.herokuapp.com/weather/${inputCity}`)
+    .then(response => response.json())
+    .then(data => displayWeather(data));
 
+  /* News */
 
+  fetch(
+    `https://newsapi.org/v2/everything?q=tesla&from=2021-11-09&sortBy=publishedAt&apiKey=${newsKey}`
```

Comment:
Cool! It seems like the issue was requesting data too far back in time, I have changed the dates in the parameters and seems to be working! 👍🏻😄